mlir/lib/Conversion/TosaToRock/TosaToRock.cpp

```diff
@@ -1120,7 +1216,8 @@ typename std::enable_if_t<
           func->getAttrOfType<SymbolRefAttr>("original_func");
       if (func::FuncOp originalFunc = dyn_cast<func::FuncOp>(
               symTable.lookupSymbolIn(rootMod, originalFuncAttr))) {
-        originalFunc.setResultAttr(0, "mhal.read_access", rw.getUnitAttr());
```

Comment:
should we add "func.setResultAttr(resNumber, rock::PrefillAttr::getMnemonic(), outputInitVal)" for original_func as well?